docker/ray-ml/Dockerfile

```diff
@@ -1,27 +1,29 @@
 ARG GPU
-FROM rayproject/ray:nightly"$GPU"
+ARG DOCKER_PREFIX
+
+FROM "$DOCKER_PREFIX"ray:nightly"$GPU"
 ARG PYTHON_MINOR_VERSION=7
 
 # We have to uninstall wrapt this way for Tensorflow compatibility
 COPY requirements.txt ./
 COPY requirements_ml_docker.txt ./
 COPY requirements_rllib.txt ./
-COPY requirements_tune.txt ./requirements_tune.txt
+# COPY requirements_tune.txt ./requirements_tune.txt
```

Comment:
Is this intentional?

Reply:
Probably, but I don't remember at this point so I'll re-enable it and figure out what it broke :)